### PR TITLE
Feature/black isort ci

### PR DESCRIPTION
**Pull request by 132ikl**

### CODE REVIEW

**Comment by 132ikl** [comment]
@dzil123 approve reee

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install autoflake black isort
+          pip install -r requirements_min.txt
```
Comment (dzil123) [inline .github/workflows/format.yml]: make it not install from requirements

**Review by dzil123 — CHANGES_REQUESTED** [review]

File: .github/workflows/format.yml (hunk 1 above)
Comment (132ikl) [reply .github/workflows/format.yml]: no

**Comment by 132ikl** [comment]
@dzil123 approve reeee

File: requirements_min.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
-aiofiles==0.4.0
```
Comment (dzil123) [inline requirements_min.txt]: stupid

**Review by dzil123 — CHANGES_REQUESTED** [review]

File: requirements_min.txt (hunk 2 above)
Comment (132ikl) [reply requirements_min.txt]: bruh

**Review by dzil123 — APPROVED** [review]

**Pull request merged**